bird1-openwrt/Makefile

```diff
@@ -99,31 +88,31 @@ $(call Package/$(PKG_LUCI_NAME)/Default)
 endef
 
 define Package/$(PKG_IPV4_UCI_NAME)/conffiles
-	/etc/config/$(BIRD4)
+  /etc/config/$(BIRD4)
 endef
 
 define Package/$(PKG_IPV6_UCI_NAME)/conffiles
-	/etc/config/$(BIRD6)
+  /etc/config/$(BIRD6)
 endef
 
 define Package/$(PKG_IPV4_UCI_NAME)/install
```

Comment:
This is why the CI is failing for you. Install section should have one tab, which was correct. Now its worse and CI complains.

Reply:
Fixing it, thanks.